### DISCUSSION TITLE
Does the docker image auto update, if not, how often should i update the docker compose file

### DISCUSSION THREAD

**Discussion by gurchgherkin** [Taxel/PlexTraktSync#2034] [Q&A] [answered]
I see the deployments are quite regular. 

Does the image auto update when new deployments are released if i am using "image: [ghcr.io/taxel/plextraktsync:latest](http://ghcr.io/taxel/plextraktsync:latest" in my docker compose file? 

If this is not the case, how often should i update the docker compose file and restart the install process? Every release of once every 3 months, 6 months, etc? 

Thanks for the help

**Comment by andcbii** [comment, marked as answer]
I'm not an expert with docker, but from what I've seen you have to manually repull the container to update it. I recommend using watchtower to auto update your containers.

https://github.com/containrrr/watchtower

**Reply by gurchgherkin** [reply]
Ok, thanks, I'll take a look at watchtower.
